planemo/galaxy/activity.py

```diff
@@ -570,10 +570,15 @@ def get_output(self, output_id):
         return self._outputs_dict[output_id]
 
     def download_output_to(self, ctx, dataset_details, output_directory, filename=None):
+        extension = dataset_details.get("file_ext", None)
+        if extension is not None:
+            extension = f".{extension}"
+        else:
+            extension = ""
```

Comment:
```suggestion
        extension = dataset_details["file_ext"]
```
this should be a mandatory attribute.